code/modules/mob/living/silicon/silicon.dm

```diff
@@ -33,6 +33,8 @@
 	silicon_mob_list |= src
 	..()
 	add_language("Ceti Basic")
+	var/datum/language/L = locate(/datum/language/common) in languages

```

Comment:
Hrm. Not fond of this, on the off chance that there is no such language assigned to the borg for reason X, Y, or Ä.

Could you query the borg's own known languages instead? Coupled with an,
```
if (L)
    default_language = L
```

To stop any runtime errors from null pointers.